python-client/giskard/core/model.py

```diff
@@ -289,15 +289,38 @@ def __init__(self,
         self.data_preprocessing_function = data_preprocessing_function
         self.model_postprocessing_function = model_postprocessing_function
 
+    def post_processing(self, raw_prediction):
+        warning_message = f"\nYour binary classification model prediction is of the shape {raw_prediction.shape}. \n" + \
```

Comment:
nitpick: `warning_message` can be initialized inside `if is_binary_classification`